rollup/rollup_sync_service/rollup_sync_service.go

```diff
@@ -225,22 +225,37 @@ func (s *RollupSyncService) parseAndUpdateRollupEventLogs(logs []types.Log, endB
 			batchIndex := event.BatchIndex.Uint64()
 			log.Trace("found new FinalizeBatch event", "batch index", batchIndex)
 
-			parentBatchMeta, chunks, err := s.getLocalInfoForBatch(batchIndex)
-			if err != nil {
-				return fmt.Errorf("failed to get local node info, batch index: %v, err: %w", batchIndex, err)
+			lastFinalizedBatchIndex := rawdb.ReadLastFinalizedBatchIndex(s.db)
+			// When failed to get lastFinalizedBatchIndex, set it's value to previous one of current batch index.
+			// It's forward compatible. (works for both before and after darwin)
```

Comment:
```suggestion
			// It's backward compatible. (works for both before and after darwin)
```

Reply:
I have fixed it by submitting a new commit manually, haven't seen this suggestion button..